packages/e2e-test-utils-playwright/src/page/canvas.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Gets the editor canvas frame.
+ *
+ * @this {import('./').PageUtils}
+ */
+export function canvas() {
+	return (
+		this.page.frames().find( ( f ) => f.name() === 'editor-canvas' ) ||
```

Comment:
on a high level it would be nice if this.page.frames() could be rewritten to frame locators: https://playwright.dev/docs/next/api/class-framelocator

since they auto wait and are more readable. But might be out of scope for the initial rewrite.